packages/design-picker/src/hooks/use-theme-designs-query.ts

```diff
@@ -0,0 +1,77 @@
+import { useQuery, UseQueryResult, UseQueryOptions } from 'react-query';
+import wpcom from 'calypso/lib/wp'; // eslint-disable-line no-restricted-imports
```

Comment:
It's a little weird to use calypso lib in the packages 🤔 Maybe we should also make it be a package

Reply:
Yeah, this is a good point. Work is underway to extract it, but it's not quite there yet. Once @tyxla is done with removing the "undocumented" methods from `lib/wp` then it'll be extractable.

[Discussed here the idea of importing `lib/wp` into Gutenboarding because it's quite a package](https://github.com/Automattic/wp-calypso/pull/58445#discussion_r760109712). It's taking a bit of a shortcut, but Calypso team didn't seem to have any worries about it into Gutenboarding early.

Reply:
Agreed. FWIW we're done with the undocumented part already: p4TIVU-9Tk-p2

As the next step, we're going to be working on packaging `lib/wp`, but it'll likely be early in 2022 as most of us are heading into holiday breaks.

Reply:
Cool! Looking forward to seeing it becoming package in early 2022 👍